pkg/gateway/handler.go

```diff
@@ -241,8 +239,12 @@ func PathOperationHandler(sc *ServerContext, handler operations.PathOperationHan
 func authorize(w http.ResponseWriter, req *http.Request, authService auth.GatewayService, perms permissions.Node) *operations.AuthorizedOperation {
 	ctx := req.Context()
 	o := ctx.Value(ContextKeyOperation).(*operations.Operation)
-	username := ctx.Value(ContextKeyUser).(*model.User).Username
-	authContext := ctx.Value(ContextKeyAuthContext).(sig.SigContext)
+	user, _ := auth.GetUser(ctx)
```

Comment:
Just in case there will be a bug and we need to identify the root cause
```suggestion
	user, err := auth.GetUser(ctx)
        if err != nil {
          o.Log(req).WithError(err).Error("failed to authorize, get user")
          _ = o.EncodeError(w, req, gatewayerrors.ErrInternalError.ToAPIErr())
          return nil
        }

```

Reply:
Done